src/components/identifiers-list.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import KeyValueLabel from './key-value-label';
+
+export default function IdentifiersList({ data }) {
+  let types = {
+    0: 'ISSN',
+    1: 'ISBN',
+    2: 'TSDID',
+    3: 'SPID',
+    4: 'EjsJournaID',
+    5: 'NewsbankID',
+    6: 'ZDBID',
+    7: 'EPBookID',
+    8: 'Mid',
+    9: 'BHM'
+  };
+
+  let subtypes = {
+    0: 'Empty',
+    1: 'Print',
+    2: 'Online',
+    3: 'Preceding',
+    4: 'Succeeding',
+    5: 'Regional',
+    6: 'Linking',
+    7: 'Invalid'
+  };
```

Comment:
Need to reach out to RM API folks to see if they can return the strings to us instead of having to map them either here or in `mod_kb_ebsco` - both options where we have to translate on our apps, whether front or back end, are very brittle.